util/fpga/cross-compiling/build-fpga-tests.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Build all Caliptra[SS] ROM and Runtime firmware for the FPGA platform.
+cargo xtask all-build --platform fpga
```

Comment:
```suggestion
cargo xtask-fpga all-build --platform fpga
```

Reply:
That is not was is documented here: https://chipsalliance.github.io/caliptra-mcu-sw/fpga.html#firmware-files What is the difference between `xtask` vs `xtask-fpga`?

Reply:
Ah looks like you answered my question here: https://github.com/chipsalliance/caliptra-mcu-sw/pull/331#discussion_r2286406242

Reply:
Both `cargo xtask` and `cargo xtask-fpga` are an alias to `cargo run -p xtask` but `cargo xtask-fpga` includes the FPGA feature flag `cargo xtask-fpga --features fpga_realtime`.

Reply:
@clundin25  do you still need the `--features fpga_realtime` flag then when running  `cargo xtask-fpga...` ?

Reply:
No sorry, fixed my comment to reflect that.